include/caf/events/led_event.h

```diff
@@ -20,22 +20,40 @@
 extern "C" {
 #endif
 
-/** @brief LED event used to change LED operating mode and color. */
+/** @brief LED event.
```

Comment:
There's no need for such a header in brief. Just write the brief from lines 25-27 here.

(Global edit for all files)

Reply:
Discussed offline.
The doc style guide does not force us either to add or not to add brief. In most places we used to add brief and include details in separate paragraph if needed. Maybe we could stick to this convention?